index.html

```diff
@@ -21,262 +21,338 @@
-      </p>
-      <div class="post-btn">
-        ul>
-          <li>
-            <button type="button">html</button>
-          </li>
-          <li>
-            <button type="button" >bootstrap</button>
-          </li>
-          <li>
-            <button type="button" >Ruby</button>
-          </li>
-      </div>
-      <li>
-        <button type="button"class="postbtn2">See Project</button>
-      </li>
-      </ul>
-    </div>
+      <img class="imageholderbig" src="bigimageplaceholder.png" alt="">
 
-    </div>
 
-    <div class="post">
-      <h1>Profesional Art Printing Data</h1>
-      <p>
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <div class="post-btn">
-        <ul>
-          <li>
-            <button type="button">html</button>
-          </li>
-          <li>
-            <button type="button" >bootstrap</button>
-          </li>
-          <li>
-            <button type="button" >Ruby</button>
-          </li>
-      </div>
-      <li>
-        <button type="button"class="postbtn2">See Project</button>
-      </li>
-      </ul>
-    </div>
 
+      <nav class="post1" id="post1-desktop">
+        <h1>Multi-Post Stories</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <nav class="post1-btn">
+          <ul>
+            <li>
+              <button class="button" type="button">css</button>
+            </li>
+            <li>
+              <button class="button" type="button">html</button>
+            </li>
+            <li>
+              <button class="button" type="button">bootstrap</button>
+            </li>
+            <li>
+              <button class="button" type="button">Ruby</button>
+            </li>
+          </ul>
 
+        </nav>
+
+
+        <button type="button" class="postb">See Project</button>
+      </nav>
+
+      <nav class="post" id="post2-desktop">
+        <h1>Profesional Art Printing Data</h1>
+
+        <p class>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <nav class="post-btn" id="post-btn-desktop1">
+          <ul>
+            <li>
+              <button class="button" type="button">html</button>
+            </li>
+            <li>
+              <button class="button" type="button">bootstrap</button>
+            </li>
+            <li>
+              <button class="button" type="button">Ruby</button>
+            </li>
+          </ul>
+
+        </nav>
+        <button class="pl" type="button" >See Project</button>
+        <button id="p" type="button">See Project</button>
+      </nav>

```

Comment:
Please don't use `nav`s for your cards, because the `<nav>` tag is used for declaring the navigational section in HTML documents.